doc/project/dev_env_and_tools.rst

```diff
@@ -315,23 +315,26 @@ Labels applicable to pull requests only
 The issue or PR describes a change to a stable API. See additional information
 in :ref:`stable_api_changes`.
 
-* ``Hot Fix``
+* ``Hotfix``
+
+Fix for an issue blocking development.
 
 * ``Trivial``
 
 * ``Maintainer``
 
+Maintainer review reqiured.
+
 * ``Security Review``
 
-Depending on the PR complexity, an indication of how long a merge should be held
-to ensure proper review. See :ref:`review process <review_time>`.
```

Comment:
This was also a change introduced in this pull request, so it's not clear why this is being done in this commit. I'm also not sure the `Licensing` label description is a good place for this to end up.

Reply:
I slightly reworded the sentence and put it as note at the end of "Labels applicable to PRs only" subsection. I think it is now more clear and reflect IMO better the original intent. What do you think, @mbolivar-nordic ?